torchvision/prototype/transforms/__init__.py

```diff
@@ -51,6 +51,7 @@
     ToDtype,
     TransposeDimensions,
 )
+from ._mixup_detection import MixupDetection
```

Comment:
Should we move this inside `transforms/_augment.py` along with the rest of MixUp/CutMix etc?